9.0/Dockerfile

```
@@ -1,5 +1,5 @@
 # Prepare a base image to use docker cache during build
-FROM debian:10-slim as base
+FROM debian:bullseye-slim as base
```

I tend to prefer numbered versions instead of codename versions.
Bullseye / 11 is in testing (not stable), is there a specific reason for this change? If not, please revert this change.
```suggestion
FROM debian:10-slim as base
```

---

Hi, thanks for the feedback.  The arm64 version of the VM could not find the correct version of libc when running under 10/buster.  I dont recall the exact error as I'm at work right now, but it was only a minor libc revision.  It's entirely possible that there was another way to resolve this without upgrading the debian version, but I was unable to do so.

Bumping to 11/bullseye worked.  I also initially tried 11-slim, but it did not exist under that tag.

FWIW, Bullseye is frozen and the release date is the end of this month, I don't know if that affects anything.

---

Just thinking aloud. Maybe it's better to have 2 Dockerfiles for Pharo 9 one for each target architecture. I find it pretty strange that the produced docker images depend on the base architecture the build is run but will use the same name. What do you think @serpi90 ?

---

IMHO, and please note that I'm relatively new to Pharo, and a longtime docker user, but far from an expert, but I like the idea of relying on the zero conf scripts within docker since they already handle the heavy lifting of platform detection, saving the Dockerfile from having to be split or handle that logic itself.

---

No problem with the version change then, will run a couple tests later and merge if i can.

@gcotelli that would not be necessary, images are tagged with architecture metadata, so you can pull from the matching architecture if it's available.

We are not building arm images (mainly because i did not research how to build and push them) but it would be a good idea, considering that pharo does work on arm

---

Is there a problem if we wait until 2021-08-14 for the official debian release?

---

No issue from me personally, I too agree it's best to stick with stable releases.  Thanks!

---

@serpi90 Maybe https://github.com/uraimo/run-on-arch-action would be useful when we tackle the generation of ARM images.